docs/introduction.rst

```diff
@@ -1,36 +1,37 @@
 Introduction
 ================
 
-This document will describe the WHP-Exchange formats used by the CCHDO for CTD and bottle.
-The WHP-exchange formats provide simplified exchange and improved readability of hydrographic data.
-WHP-exchange data files carry the essential information from CTD and water sample profiles.
-WHP-exchange is a rigorously-described comma-delimited (csv) format designed to ease data exchange and simplify data import.
+This document will describe the WHP-Exchange formats used by the CCHDO for CTD and bottle data where CTD files store conductivity, temperature, and depth profiles and bottle files store water sample profiles.
+WHP-Exchange is a rigorously-described comma-delimited (csv) format designed to ease data exchange and simplify data import.
+WHP-Exchange data files carry the essential information from CTD and bottle profiles.
+The WHP-Exchange formats provide improved readability and a simplified exchange of hydrographic data.
 
 
 Overview
 --------
-The WHP-exchange bottle and CTD data formats include these features:
+The WHP-Exchange CTD and bottle data formats include these features:
 
 * UTF-8 Encoded
 * Spreadsheet-like
 * Comma-delimited values (csv)
 * No special meaning to blank/empty spaces
-* Station information in every line in the file (bottle) or in the top lines in each file (CTD)
-* Only one missing data value defined for all parameters
+* Station information added in the top lines for a CTD file
+* Station information added in every line for a bottle file
+* Only one missing data value defined for all parameters **(What does this mean?)**
```

Comment:
This means there are no parameter specific fill values, so if you see a `-999` it is a fill value no matter where it occurs in the document.